Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/download/DownloadDiagnosisKeysTask.kt

```diff
@@ -70,18 +73,30 @@ class DownloadDiagnosisKeysTask @Inject constructor(
             checkCancel()
 
             // RETRIEVE RISK SCORE PARAMETERS
-            val exposureConfiguration = appConfigProvider.getAppConfig().exposureDetectionConfiguration
+            val exposureConfig: ExposureDetectionConfig = appConfigProvider.getAppConfig()
 
             internalProgress.send(Progress.ApiSubmissionStarted)
             internalProgress.send(Progress.KeyFilesDownloadStarted)
 
             val requestedCountries = arguments.requestedCountries
-            val availableKeyFiles = getAvailableKeyFiles(requestedCountries)
+            val keySyncResult = getAvailableKeyFiles(requestedCountries)
             checkCancel()
 
-            val totalFileSize = availableKeyFiles.fold(0L, { acc, file ->
-                file.length() + acc
-            })
+            val trackedDetections = enfClient.latestCalculations().first()
```

Comment:
Let's not be lazy and call this `trackedExposureDetections` 😉

Reply:
It's not about "being lazy".
Horizontal space is limited, identiation and wrap around makes things less readable, readers attention span is limited.
It's about providing the reader with the right amount of information to understand the code, and not overburden them with unnecessary noise.